### ISSUE TITLE
Show all tags in a drop down list on click.  Similar to the MODX Multi Select TV

### ISSUE THREAD

**Issue by neoneddy** [goldsky/SmartTag#34]
CLient is confused by having to type.  And it's true.  WE have mixed behavior.  Some fields have a drop down, others no, but both offer type ahead.

**Comment by goldsky** [comment]
ehm, the purpose of this tagging type is to NOT showing the existing.
This extra was built at the first place to store thousands of the tags.
So Avoid ing to show them at once was the original purpose.

**Comment by Parthian** [comment]
Hi Goldsky.

Enjoyed using SmartTag. Particularly the conversion tools. However, I share the concern of neoneddy. I expected to see a list of Tags for Editors to select as with AutoTag and Tag TV Types. This to help a user decide which Tag to use.

As is, out of sight out of mind. Appreciate your comment that this package is for massive tag lists where showing them in the resource Template Variable area wouldn't work. Maybe consider a setting where if there are less than 20 or 30 tags they can appear as with AutoTags with the cute little label or something. Best of both.

Just a suggestion

Stuart

**Comment by neoneddy** [comment]
Yeah coming back around to this...  we've used auto tag a few times where smarttag is a superior technical solution, but the users prefer auto tag better.

Thank you for all the hard work.
